### PR TITLE
A few tweaks to #10581

### PR DESCRIPTION
**Pull request by alextd**
**Bug fix/grammar fix**

## Summary
Fix grammar, adjust code line length, fix math, use "buy" or "load" as appropriate.

(As I previously touched this area of code, I saw this change, and tweaking it for correctness)

## Testing Done
The code is simple enough to not need testing.

### CODE REVIEW

File: source/OutfitterPanel.cpp [hunk 1]
```diff
@@ -385,14 +385,14 @@ ShopPanel::BuyResult OutfitterPanel::CanBuy(bool onlyOwned) const
 		int64_t credits = player.Accounts().Credits();
 
 		if(cost > credits)
-			errors.push_back("You do not have enough money to buy this outfit, you need a further " +
+			errors.push_back("You do not have enough money to buy this outfit. You need a further " +
```
Comment (mOctave) [inline source/OutfitterPanel.cpp:388]: Semicolon could also work here.

File: source/OutfitterPanel.cpp [hunk 2]
```diff
@@ -475,7 +475,8 @@ ShopPanel::BuyResult OutfitterPanel::CanBuy(bool onlyOwned) const
 		return errors[0];
 	else
 	{
-		string errorMessage = "There are several reasons why you cannot buy this outfit:\n";
+		string errorMessage = "There are several reasons why you cannot " +
+			string(onlyOwned ? "load" : "buy") + " this outfit:\n";
```
Comment (mOctave) [inline source/OutfitterPanel.cpp:479]:
```suggestion
			string(onlyOwned ? "install" : "buy") + " this outfit:\n";
```

To fit with the rest of this section. Also, `load` is ambiguous between installation, carriage as cargo, and loading from an external file.

**Review by mOctave — COMMENTED** [review]
Generally looks good to me; just a few things that could be tweaked.

I would strongly suggest testing this, since no one-liner is ever so small as to be unable to irrevocably break the universe.

File: source/OutfitterPanel.cpp (hunk 2 above)
Comment (alextd) [reply source/OutfitterPanel.cpp:479]: Oh I just used the same "load" phrase that was elsewhere. The outfitter is always confusing whether a button installs, moves, or buys, and from store, cargo or storage ...

Comment (mOctave) [reply source/OutfitterPanel.cpp:479]: Huh...
I think where that snippet was used (OutfitterPanel.cpp, line 407), it was referring to cargo. Maybe something to change in a separate PR there.

File: source/OutfitterPanel.cpp (hunk 1 above)
Comment (xobes) [reply source/OutfitterPanel.cpp:388]: I re-broke this in #10743, and have just fixed it locally.

File: source/OutfitterPanel.cpp (hunk 2 above)
Comment (xobes) [reply source/OutfitterPanel.cpp:479]: #10743 makes this OBE

**Pull request merged**